src/api/api.cc

```diff
@@ -10796,7 +10796,7 @@ typedef char* (CommandCallbackRaw)(const char* params);
   Macro(RecordReplayJSONFree, (void*))                                        \
   Macro(RecordReplayOnAnnotation, (const char* kind, const char* contents))   \
   Macro(RecordReplayAddPossibleBreakpoint,                                    \
-        (int line, int column, const char* function_id, int offset))          \
+        (int line, int column, const char* function_id, int rank))          \
```

Comment:
`rank` is a weird term to use for this value, can we change this to `function_index` and similar here and in the rest of the patch?

Reply:
The problem with using the word `function` is that its not always a `function`. Sometimes its at the top/file/script/module level 👀

Reply:
But then again, we aready use `function_id`, so we might as well 🤷‍♀️

Reply:
Using "function" here is at least consistent with how V8 uses it -- a Script is an entire file's (or <script> block's) worth of code, while a SharedFunctionInfo is used for both the functions and the top level scripts, modules etc within that script.